### ISSUE TITLE
EF Core repeatedly creating same migration

### ISSUE THREAD

**Issue by atmike** [dotnet/efcore#23731]
Hello,
I'm sorry to bother you with my problem, but I've been trying to solve it for half a year now and I'm not getting anywhere.
We are currently working on a solution for one of our customers where we are developing several modules. Each module has its own DBContext with which it can use the tables and also create migrations. This works perfectly. But now it is so, that most modules still have to access the tables of a base module and also include them in their own selects. 
Here we proceed in such a way, that we include the entities of the basis module into the own DBContext as View => 
```C#
modelBuilder.Entity<ProjectSetting>().ToView("MASTER_PROJECTSETTING");
```
which also works very well.

Our problem is that the migartion does not ignore these views but takes everything in every migartion you run over and over again.

I have already gone through all the possible solutions to solve this problem. 
- Primary keys must not be a uint
- create a migration once without changes, delete everything 
- ...

But unfortunately none of them works. What I need is the possibility to exclude the views complet from the migartion

Does anyone else have an idea how I can find out what problem the migration has?

thank you for advise and regards

Michael

EF Core version: 3.1.4
Database provider: (e.g. Microsoft.EntityFrameworkCore.SqlServer)
Target framework: (e.g. .NET 3.1)
Operating system: Windows 10 latest updates
IDE: (e.g. Visual Studio 2019 16.7.3)

**Comment by anranruye** [comment]
Does any other entities refer to these entities mapped to view?  I guess this can be the cause. Otherwise,  this situation should not happen.

**Comment by ajcvickers** [comment]
@atmike

> Our problem is that the migartion does not ignore these views but takes everything in every migartion you run over and over again.

There is nothing here that should have this behavior. However, we have fixed several bugs in this area for 5.0. Can you try updating to EF Core 5.0.1 and report back if you still see this issue.

**Comment by ajcvickers** [comment]
**EF Team Triage:** Closing this issue as the requested additional details have not been provided and we have been unable to reproduce it.

*BTW this is a canned response and may have info or details that do not directly apply to this particular issue. While we'd like to spend the time to uniquely address every incoming issue, we get a lot traffic on the EF projects and that is not practical. To ensure we maximize the time we have to work on fixing bugs, implementing new features, etc. we use canned responses for common triage decisions.*
